pkg/standalone/docker.go

```diff
@@ -127,3 +127,13 @@ func imageFileName(image string) string {
 	filename = strings.ReplaceAll(filename, ":", "-")
 	return filename
 }
+
+func IsImageInGHCR(imageName string) bool {
+	args := []string{
+		"image",
+		"inspect",
+		imageName,
```

Comment:
This is only for local image inspection .... 

try using `docker pull <imageName>`